From 5609d06647c049350f3a4898e510aabcf1f4e2b7 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Fri, 21 Jun 2024 14:26:51 +0200
Subject: [PATCH 0930/1695] drivers: stm32_rif: add CIDCFGR macros

Add macros for SCID and SEMWL.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git f5cbd895b400acc7cce81cad5b3e5c74d8ce4f7b drivers: stm32_rif: add CIDCFGR macros]
Change-Id: I313ce5d01e6e215be9b79cf787975f2f067f5874
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/389464
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Etienne CARRIERE <etienne.carriere@foss.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 core/include/drivers/stm32_rif.h | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/core/include/drivers/stm32_rif.h b/core/include/drivers/stm32_rif.h
index 65e61de52..2f74c04aa 100644
--- a/core/include/drivers/stm32_rif.h
+++ b/core/include/drivers/stm32_rif.h
@@ -16,8 +16,10 @@
  * CIDCFGR register
  */
 #define _CIDCFGR_CFEN			BIT(0)
+#define _CIDCFGR_SCID_SHIFT		U(4)
 #define _CIDCFGR_SEMEN			BIT(1)
-#define _CIDCFGR_SEMWL(x)		BIT(SEMWL_SHIFT + (x))
+#define _CIDCFGR_SEMWL_SHIFT		U(16)
+#define _CIDCFGR_SEMWL(x)		BIT(_CIDCFGR_SEMWL_SHIFT + (x))
 
 /*
  * SEMCR register
-- 
2.47.2

